src/redux/categories/categories.js

```diff
@@ -0,0 +1,22 @@
+// Actions
+const CHECK_STATUS = 'CHECK_STATUS';
+
+// reducer
+export default function categories(state = [], action) {
+  switch (action.type) {
+    case CHECK_STATUS:
+      return {
+        ...state,
+        status: 'Under construction',
+      };
```

Comment:
- [ ] Kindly change the implementation of the `CHECK_STATUS` case to be:
```
return 'Under construction'
```
instead of returning the object as per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/add_reducers_an_actions.md#:~:text=For%20the%20check%20status%20action%20return%20a%20string%20%22Under%20construction%22.)